src/foam/nanos/auth/CountryService.java

```diff
@@ -72,7 +71,7 @@ protected void notifySupport(String query) {
     emailMessage.setSubject ("Unknown Country");
     emailMessage.setBody    ("User just added an unknown country " + query);
```

Comment:
This is legacy code, but this is where I think we should use notification templates so we don’t have hard coded emails in code.   Other parts of the system are using a notification template ‘NOC’, which would be configured per installation and had an appropriate email(s).

Reply:
no ticket at the moment, just noticed it, and yes, ok, if in another PR.